Apps/frontend/src/components/Slider.svelte

```diff
@@ -14,6 +14,6 @@
 <div class={className} data-cy={dataCy}>
   <input {id} type="range" bind:value on:input={onInput} {min} {max} {step} />
   {#if calculateDisplayedValue !== undefined}
-    <span>{calculateDisplayedValue(value)}</span>
+    <div style="display: inline;">{calculateDisplayedValue(value)}</div>
```

Comment:
Please no inline styling, all styling should be in `app.scss`. On the same note, why did you change `span` to `div`?

Reply:
Moved to css class